hazenlib/ACRObject.py

```diff
@@ -5,12 +5,14 @@
 
 
 class ACRObject:
-    def __init__(self, dcm):
-        self.dcm = dcm
-        self.images, self.dcm = self.sort_images()
+    def __init__(self, dcm_list):
+        self.dcm_list = dcm_list
+        self.images, self.dcms = self.sort_images()
+        self.pixel_spacing = self.dcms[0].PixelSpacing
         self.orientation_checks()
         self.rot_angle = self.determine_rotation()
         self.centre, self.radius = self.find_phantom_center()
+        self.mask_image = self.get_mask_image(self.images[6])
```

Comment:
rename some variables and functions that better describe their purpose
create a few reusable parameters that are often used in calculations (pixel_spacing)